pdc/apps/release/models.py

```diff
@@ -124,6 +127,8 @@ class ProductVersion(models.Model):
     product             = models.ForeignKey(Product)
     product_version_id  = models.CharField(max_length=200)
 
+    masked_push_targets = models.ManyToManyField(PushTarget)
+
```

Comment:
As masked_push_targets is sub-set of allowed_push_targets, it's better to add a docstring about this.
Because it isn't showed on portal or output, just write in codes, maybe we will forget the meaning in the future.

Reply:
You're right, but I'm not sure where to add the docstring. Maybe I could subclass the field. I'll try to do something with the repeated code in `allow_push_targets` properties too.

Reply:
I've created abstract model class with the new field to remove the duplicate code and added explanatory docstring. I spent quite some time trying to figure out how to subclass ManyToManyField instead, but failed -- it doesn't seems to call the documented methods (Django documentation fails to mention this).